### DISCUSSION TITLE
Window moving keyboard shortcut by fourths

### DISCUSSION THREAD

**Discussion by andripeetso** [rxhanson/RectanglePro-Community#513] [General]
I just switched from 2x 27" screens to 1x 49"

However, before my most used shortcut was the alt+option and left & right keys to snap window to halves on either monitor.

But now with a single ultrawide this doesn't work and I would need four zones.

How can I map this shortcut work for fourths?

**Comment by rxhanson** [comment]
There are two options:
1. Use the fourths shortcuts and repeat the keyboard shortcuts for first and last fourth, which will cycle through the fourths provided you don't have `Repeated commands` set to `do nothing` in the General tab. This isn't exactly what you want, but would likely be pretty quick to get the hang of. 
2. You can make a shortcut in "Custom Size & Position" like the following:

![image](https://github.com/rxhanson/RectanglePro-Community/assets/13651296/d497a3f6-fc37-4c56-bc2e-0b7da1f5e843)
